packages/docusaurus-plugin-content-docs/src/sidebars/index.ts

```diff
@@ -89,7 +89,16 @@ export async function loadSidebarsFileUnsafe(
   }
 
   // We don't want sidebars to be cached because of hot reloading.
-  return importFresh(sidebarFilePath);
+  const sidebars = jiti(__filename, {
+    cache: false,
+    interopDefault: true,
+    debug: true,
+  })(sidebarFilePath);
```

Comment:
This does not look like a drop-in replacement for `importFresh` unfortunately.

Calling it multiple times will return the same result despite changing the underlying file.

Reply:
Hi. You can set `requireCache: false` in order to disable (runtime) cache. `cache` option is for filesystem transform cache (and can be even kept enabled for better performance. you still will get a fresh value each time).

Please let me know if could help further on this :)

Reply:
Also, we are working on new API `jiti.import()` (releasing by next week) if PR was delayed until then would be nice that you consider the migration for better future compatibility. It can bring support more native ESM compatibilities.

Reply:
Thanks a lot @pi0 that's working. I saw `requireCache` in an older issue but thought it was an older removed option because it's not in v1 or v2 docs. But it's in the type and seems to work as I expect 👍 

Regarding `jiti.import()` (https://github.com/unjs/jiti/issues/174?) I assume you want to say v2.0 is being released soon? 

On my side I didn't expect to migrate away from `importFresh` in Docusaurus v3 but as jiti seems to work that would be a awesome improvement for our users so I'm already delaying a bit our official release to incorporate this PR 😄 I'd prefer to not be an early adopter of jiti v2 this time, for timing reasons it's preferable if we upgrade for Docusaurus v4 (or in a minor version later, if not considered as a breaking change)

Reply:
Ah awesome!

`jiti.import()` API will be also released as a backward compatible stub for v1 for future compatibility so you can migrate earlier and only bump the major version of jiti later. (i am on holidays with family RN but might find some time to do it earlier). v2 is not expected that soon.

Reply:
Great, thanks :)

From what I understand currently Jiti is able to load modules synchronously but import will be async so it's probably better to assume async usage in the future?